newIDE/app/src/SceneEditor/index.js

```diff
@@ -1403,7 +1422,10 @@ export default class SceneEditor extends React.Component<Props, State> {
                   })
                 }
                 getAllObjectTags={this._getAllObjectTags}
-                ref={objectsList => (this._objectsList = objectsList)}
+                ref={
+                  // $FlowFixMe Make this component functional.
+                  objectsList => (this._objectsList = objectsList)
+                }
```

Comment:
Should we address this?

Reply:
Is it better with `ref={this._objectsList}`?